client/webserver/site/src/js/markets.ts

```diff
@@ -632,7 +677,31 @@ export default class MarketsPage extends BasePage {
     const hasWallets = base && app().assets[base.id].wallet && quote && app().assets[quote.id].wallet
 
     if (feePaid && assetsAreSupported && hasWallets) {
+      const lot = '1'
+      const lotSize = String(this.market.cfg.lotsize / this.market.baseUnitInfo.conventional.conversionFactor)
+      const rateStep = String(this.market.cfg.ratestep / this.market.rateConversionFactor)
+      const rate = this.defaultRate()
+
+      // Reset limit-order form inputs to defaults.
+      page.lotField.min = lot
+      page.lotField.step = lot
```

Comment:
It's not clear to me that setting `min` and `step` actually do anything for us. If I comment these out, it doesn't seem to change my ability to type, say, a decimal number in the lots input.

Reply:
Same.  Browser ignores it except for changing the little hover message.

Reply:
> except for changing the little hover message

yes, that's exactly the reason to change this, because otherwise the message shown doesn't make sense to me

Reply:
I couldn't find way to remove that hover message, so having `min` & `step` is probably best trade-off here to at least correct what it displays.

And there is another reason as well, I just remembered, which is handling up/down key-press handling correctly:
- for `lotField ` the values I'm setting just happen to match the defaults (but sometimes it is worth to specify defaults explicitly so we know what they are, and that it is intentional that we are using these defaults) 
- for `qtyField` in makes sense to increment/decrement in, say, 20s than by 1 when `lotSize=20` (when using up/down keys on keyboard)
- and similar applies for `rateField` ^

Added [commit](https://github.com/decred/dcrdex/pull/1921/commits/3d5afd10867b39c237c2e4a5eedcb95085f06b83) to document some reasoning for this.